File: src/org/ggp/base/util/concurrency/ConcurrencyUtils.java
package org.ggp.base.util.concurrency;

class ConcurrencyUtils(object):
	/**
	 * If the thread has been interrupted, throws an InterruptedException.
	 */
    def static void checkForInterruption() throws InterruptedException {
        if (Thread.currentThread().isInterrupted())
            throw new InterruptedException();

File: ggpy/cruft/autocode/ConcurrencyUtils.py
#!/usr/bin/env python
""" generated source for module ConcurrencyUtils """
# package: org.ggp.base.util.concurrency
class ConcurrencyUtils(object):
    """ generated source for class ConcurrencyUtils """
    # 
    # 	 * If the thread has been interrupted, throws an InterruptedException.
    # 	 
    @classmethod
    def checkForInterruption(cls):
        """ generated source for method checkForInterruption """
        if Thread.currentThread().isInterrupted():
            raise InterruptedException()

